docs/general/ddl/data-types.rst

```diff
@@ -561,6 +565,37 @@ Time zone syntax as defined by `ISO 8601 time zone designators`_.
     +-----------------------+
     SELECT 1 row in set (... sec)
 
+.. _date-data-type:
+
+date
+-------------------
+
+A string literal that contain a date will be
+converted to UTC date.
+
+::
+
+    cr> select '2021-03-09'::date as cd;
+    +---------------+
+    |            cd |
+    +---------------+
+    | 1615248000000 |
+    +---------------+
+    SELECT 1 row in set (... sec)
+
+
+Date also accepts a ``bigint`` representing UTC milliseconds since
+the epoch.
+
+::
+
+    cr> select 1615248000000::date as cd;
+   +---------------+
+    |            cd |
+    +---------------+
+    | 1615248000000 |
+    +---------------+
+    SELECT 1 row in set (... sec)
```

Comment:
I'd remove this example if you take my suggestion above which already mention the support of numeric->date casts.

Reply:
Done. I removed string casting example `select '2021-03-09'::date` as well as it's also mentioned in suggestion.

Reply:
I think keeping the string cast example would still be valuable (also this may be the common use case).

Reply:
done